packages/components/app/styles/components/stepper/index.scss

```diff
@@ -1,2 +1,2 @@
-@use './indicator/step.scss';
-@use './indicator/task.scss';
\ No newline at end of file
+@use './step-indicator.scss';
```

Comment:
we could do this in a follow-up PR, but it _seems_ like `/.step/indicator.scss` is the likely structure we'll want here eventually

Reply:
That's the structure I had initially, but it threw a Sass error for a duplicative namespace 🤔

Reply:
sounds...sassy 🙃

Reply:
i see what you mean. something we can dig into later 👍